Documentation error: Node.js version prerequisite not properly mentioned

The main README specifies Node.js v16.19.0 as the required version for local development:
>Node.js: Version 16.19.0 is required. Use Node Version Manager (NVM) for easy switching.

However, when running the docs website inside the docs folder, the following error appears if Node.js version is below 18:
```bash
[ERROR] Minimum Node.js version not met :(
[INFO] You are using Node.js v16.19.0, Requirement: Node.js >=18.0.
```
This inconsistency can confuse developers trying to set up the environment.

## Suggested fixes:
- Clarify in the docs README that the `docs` folder requires Node.js 18 or higher.
-  Alternatively, update dependencies to align Node.js version requirements across the repository.

Improving this will enhance the onboarding experience and reduce setup errors.

--- a/packages/docs/README.md
+++ b/packages/docs/README.md
@@ -1,17 +1,29 @@
 # EmbeddedChat Documentation
 
-This is the official documentation website of EmbeddedChat
+This is the official documentation website of EmbeddedChat.
+
+> **Node.js Version Requirement**
+>
+> The `docs/` folder requires **Node.js v18 or higher** to run correctly.
+> If you’re using a lower version (e.g., v16.19.0 from other parts of the monorepo), you may encounter errors.
+>
+> Use [NVM](https://github.com/nvm-sh/nvm) to install and switch to the correct version:
+>
+> ```bash
+> nvm install 18
+> nvm use 18
+> ```
 
 ### Installation
 
 ```
-$ yarn install
+ yarn install
 ```
 
 ### Local Development
 
 ```
-$ yarn dev
+ yarn dev
 ```
 
 This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
@@ -19,7 +31,7 @@ This command starts a local development server and opens up a browser window. Mo
 ### Build
 
 ```
-$ yarn build
+ yarn build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.

